lib/src/chatbox.dart

```diff
@@ -91,6 +91,7 @@ class ChatBox extends StatefulWidget {
   final bool? showChatHeader;
   final TranslationToggle? showTranslationToggle;
   final String? theme;
+  final Map<String, dynamic>? custom;
```

Comment:
I'm not convinced about calling this property `custom`, because in the future we might implement something custom for the chatbox, and that will cause confusion.

Given that Dart doesn't implement union types (the worst omission in the language ever), I'd rather have a `themeOptions` property, of type `ThemeOptions`.
If both `theme` and `themeOptions` are passed, `themeOptions` would take precedence over theme.